src/diffusers/modeling_flax_utils.py

```diff
@@ -272,6 +275,7 @@ def from_pretrained(
         ```"""
         config = kwargs.pop("config", None)
         cache_dir = kwargs.pop("cache_dir", DIFFUSERS_CACHE)
+        ignore_mismatched_sizes = kwargs.pop("ignore_mismatched_sizes", False)
```

Comment:
Can we remove this for now? I don't think we should allow mismatched sizes for now

Reply:
made it stricter https://github.com/huggingface/diffusers/pull/513/commits/73e0bc692c5761e55faff39c80a26d7a3cfc748c